aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -103,6 +103,24 @@ void logical_xor_kernel(TensorIterator& iter) {
     });
 }
 
+void lt_kernel(TensorIterator& iter) {
+  if (iter.dtype() == ScalarType::Bool) {
+    AT_DISPATCH_ALL_TYPES_AND2(kBool, kBFloat16, iter.input_dtype(), "lt_cpu", [&]() {
+      cpu_kernel(iter,
+       [=](scalar_t a, scalar_t b) -> bool {
+         return a < b;
+       });
+    });
+  } else {
+    AT_DISPATCH_ALL_TYPES_AND(kBFloat16, iter.dtype(), "lt_cpu", [&]() {
```

Comment:
Is `bfloat16` a CPU-only type?

Reply:
Based on this: https://github.com/pytorch/pytorch/pull/25998#discussion_r326769207

Reply:
I thought it corresponds to kHalf

Reply:
I have no clue what bfloat16 is, but I think it is different from `kHalf`. @izdeby would know

Reply:
I believe it's this: https://en.wikipedia.org/wiki/Bfloat16_floating-point_format

It's similar to half (in that it's a two-byte floating point type), but the representation is different from kHalf.

Reply:
My real question was: "Why do we only allow comparisons for bfloat16 in CPU but not CUDA?" I think the answer to that is that we don't have bfloat16 support on CUDA.